v2/jdbc-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/templates/JdbcToBigQueryIT.java

```diff
@@ -320,7 +320,11 @@ public void testPostgresReadWithPartitions() throws IOException {
         postgresResourceManager,
         false,
         false,
-        config -> config.addParameter("table", testName).addParameter("partitionColumn", ROW_ID));
+        config ->
+            config
+                .addParameter("table", testName)
+                .addParameter("partitionColumn", ROW_ID)
+                .addParameter("partitionColumnType", "long"));
```

Comment:
would the test fail if do not add "partitionColumnType" line. We should make sure it does not introduce breaking change